pkg/tenant/api/v1/api.go

```diff
@@ -187,11 +187,15 @@ type TenantPage struct {
 }
 
 type Member struct {
-	ID       string `json:"id" uri:"id"`
-	Name     string `json:"name"`
-	Role     string `json:"role"`
-	Created  string `json:"created,omitempty"`
-	Modified string `json:"modified,omitempty"`
+	ID           string `json:"id" uri:"id"`
+	Email        string `json:"string"`
+	Name         string `json:"name"`
+	Role         string `json:"role"`
+	Status       string `json:"status"`
+	Created      string `json:"created,omitempty"`
+	Modified     string `json:"modified,omitempty"`
+	DateAdded    string `json:"date_added,omitempty"`
+	LastActivity string `json:"last_activity,omitempty"`
```

Comment:
Is `LastActivity` just `LastLogin`? I think that's a great field to add, but just wanted to make sure I understand how it's going to be used.

Reply:
Yes, during the meeting we had I confirmed that `LastActivity` is the same as `LastLogin`. The design uses `Last Activity` as a label, so I used the same wording here.